digitaloceanmodule.php

```diff
@@ -331,6 +331,30 @@ public function getPackageFields($vars = null) {
         return $fields;
     }
 
+	public function getallImagesDropdown($package) {
+        $row = $this->getModuleRow($package->module_row);
+        $api = $this->getApi($row->meta->apiKey);
+        $result = $api->getlongGetResults("images");
+        $dp = array();
+		foreach ($result->images as $key => $value)
+		{
```

Comment:
The rest of the class places braces on the same line as the condition. You should follow the same syntax.